app.py

```diff
@@ -23,6 +23,10 @@ def __init__(self):
         flowbox.set_max_children_per_line(30)
         flowbox.set_selection_mode(Gtk.SelectionMode.NONE)
 
+        shift_label = Gtk.Label()
+        shift_label.set_text("Choose the Shift")
+        self.shift_input = Gtk.SpinButton.new_with_range(1, 25, 1)
```

Comment:
Adds the spin button with range so that we can step by 1 and limit the options from only 1-25 to account for the length of the alphabet.